server.go

```diff
@@ -1100,6 +1100,9 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 			// In case the user has explicitly specified
 			// a default value for the number of
 			// confirmations, we use it.
+			if chainCfg.SkipChannelConfirmation {
```

Comment:
Ah I see now, since the _receiver_ is the on that sends back how many confirmations are required, it's a config value. In theory, it could also be implemented as a channel acceptor as well, which would sort of help with risk mitigation, as the user could only whitelist certain nodes as being acceptable to receive a zero conf channel from.